server.js

```diff
@@ -578,6 +578,41 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/coverage\.(svg|png|gif|jpg|json)
       });
     }));
 
+// Coverity integration
+camp.route(/^\/coverity\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var projectId = match[1]; // eg, `3997`
+  var format = match[2];
+  var url = 'https://scan.coverity.com/projects/' + projectId + '/badge.json';
+  var badgeData = getBadgeData('coverity', data);
+  request(url, function(err, res, buffer) {
+    if (err != null) {
+      console.log(err);
```

Comment:
Could you have something like `badgeData.text[1] = 'inaccessible';` instead, and remove the log?